src/state/tokenPrices/hooks.ts

```
@@ -68,6 +68,8 @@ const useTokenPricesLocal = (
 
           dispatch(updatePrices(formattedPrices))
         }
+      } catch (e) {
+        // empty
```

Is this intended empty? Because this is useless

---

There's no catch before. So I just added an empty catch :V